ai_workloads/kueue/monitoring-pending-workloads.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="monitoring-pending-workloads-install-kueue"]
+= Monitoring pending workloads 
+:context: monitoring-pending-workloads
+
+toc::[]
+
+{kueue-name} provides the `VisibilityOnDemand` feature to monitor pending workloads. This feature provides the ability for batch administrators to monitor the pipeline of pending jobs, and help users to estimate when their jobs will start.
+
+.Prerequisites 
+
+* A Kubernetes cluster is running.
+* You have cluster administrator permissions.
+* The {kueue-name} Operator is installed on your cluster, and you have created a `Kueue` custom resource (CR).
+* You have installed the {oc-first}. 
+* The {oc-first} has communication with your cluster.
+* Kueue is installed in version v0.6.0 or later.
```

Comment:
We don't need to mention this because the version of the Kueue operand that is installed is already defined internally in the kueue-operator 1.2.